test/countries/test_chile.py

```diff
@@ -9,100 +9,371 @@
+        self.assertHoliday(*[f"{year}-07-16" for year in range(2007, 2050)])
+        self.assertNoHoliday(*[f"{year}-07-16" for year in range(1915, 2007)])
+
+    def test_assumption_of_mary(self):
+        self.assertHoliday(*[f"{year}-08-15" for year in range(1915, 2050)])
+
+    def test_national_liberation(self):
+        self.assertHoliday(*[f"{year}-09-11" for year in range(1981, 1999)])
+        self.assertNoHoliday("1980-09-11", "1999-09-11")
+        for year in range(1915, 1981):
+            self.assertNotIn(
+                "Día de la Liberación Nacional",
+                ";".join(Chile(years=year).values()),
+            )
+        for year in range(1999, 2050):
+            self.assertNotIn(
+                "Día de la Liberación Nacional",
+                ";".join(Chile(years=year).values()),
+            )
+
+    def test_national_unity(self):
+        self.assertHoliday(
+            "1999-09-06",
+            "2000-09-04",
+            "2001-09-03",
+        )
+        self.assertNoHoliday("1998-09-07", "2002-09-02")
+        for year in range(1915, 2050):
+            if year not in {1999, 2000, 2001}:
+                self.assertNotIn(
+                    "Día de la Unidad Nacional",
+                    ";".join(Chile(years=year).values()),
+                )
+
+    def test_independence_holidays(self):
+        self.assertHoliday(*[f"{year}-09-18" for year in range(1915, 2050)])
+        self.assertHoliday(*[f"{year}-09-19" for year in range(1915, 2050)])
+        self.assertHoliday(
+            "2007-09-17",
+            "2012-09-17",
+            "2013-09-20",
+            "2018-09-17",
+            "2019-09-20",
+            "2021-09-17",
+            "2024-09-20",
+        )
+        self.assertHoliday(*[f"{year}-09-20" for year in range(1932, 1945)])
+
+    def test_columbus_day(self):
+        for year in range(1922, 2000):
+            if year != 1973:
+                self.assertHoliday(f"{year}-10-12")
+                self.assertIn(
+                    "Día de la Raza [Columbus day]",
+                    ";".join(Chile(years=year).values()),
+                )
+            else:
+                self.assertNotIn(
+                    "Día de la Raza [Columbus day]",
+                    ";".join(Chile(years=year).values()),
+                )
+        self.assertHoliday(
+            "2000-10-09",
+            "2005-10-10",
+            "2010-10-11",
+            "2015-10-12",
+            "2019-10-12",
+            "2020-10-12",
+            "2021-10-11",
+            "2022-10-10",
+            "2023-10-09",
+        )
+        for year in range(2000, 2020):
+            self.assertIn(
+                "Día del Respeto a la Diversidad",
+                ";".join(Chile(years=year).values()),
+            )
+        for year in range(2020, 2050):
+            self.assertIn(
+                "Día del Descubrimiento de dos Mundos",
+                ";".join(Chile(years=year).values()),
+            )
```

Comment:
@arkid15r, maybe you can suggest a better code to check specific holiday presence/absence in year?

Reply:
With `get_named `, test for **presence** "Ascensión del Señor" will look like
```
self.assertNotEqual(Chile(years=year).get_named("Ascensión del Señor"), [])
```
?

Reply:
`self.assertTrue(Chile(years=year).get_named("Ascensión del Señor"))` would also work.
It's up to you, these tests are going to be refactored soon anyway.

Reply:
Ok, thank you. So for now I'll leave it as is.